src/test/scala/com/cognite/sdk/scala/v1/DataSetsTest.scala

```diff
@@ -0,0 +1,96 @@
+package com.cognite.sdk.scala.v1
+import java.time.Instant
+
+import com.cognite.sdk.scala.common._
+import com.cognite.sdk.scala.common.SetValue
+
+class DataSetsTest extends SdkTestSpec {
+
+  it should "able to create datasets if doesn't exists" in {
```

Comment:
Not sure I'm reading this test right - it checks for the existence of a data set, and doesn't write it if it exists? Why does it depend on the existence/non-existence of a data set with externalIdPrefix `testDataset1-`?

Reply:
Yeah so the problem with keep on creating datasets was we can't delete them.
So, yeah I can filter all the datasets and check if it exists and then add or do an error handling and then add. 
We just need to check the create functionality.  It actually doesn't matter on filtering wt to externalId. I just did that because we would getting only one result !
I will change it.

Reply:
I'm not sure if it's a problem that we just create new ones on each test run. There's no limit to how many data sets we can have, I think? And eventually there will be delete support I guess.